src/App.js

```diff
@@ -1,12 +1,198 @@
-import React from 'react';
+import React, {useState} from 'react';
 import './App.css';
+import Button from "./Atributes/Elements/Button";
+import {addOne, minOne, resetCount} from "./Atributes/Functions/Functions";
+import FruitComponent from "./Atributes/Elements/FruitComponenet";
+import Field from "./Atributes/Elements/Field";
+import Radio from "./Atributes/Elements/Radio";
 
 function App() {
+
+    const [bananas, setBananas] = useState(0);
+    const [strawberries, setStrawberries] = useState(0);
+    const [apples, setApples] = useState(0);
+    const [kiwis, setKiwis] = useState(0);
+
+    const [firstname, setFirstname] = useState("");
+    const [lastname, setLastname] = useState("");
+    const [age, setAge] = useState(0);
+    const [zipcode, setZipcode] = useState("");
+    const [delivery, setDelivery] = useState("");
+    const [time, setTime] = useState("");
+    const [text, setText] = useState("");
+    const [check, setCheck] = useState(true);
+
+    function resetAll() {
+        setBananas(0);
+        setStrawberries(0);
+        setApples(0,)
+        setKiwis(0);
+        setFirstname("");
+        setLastname("");
+        setAge(0);
+        setZipcode("");
+        setDelivery("every day");
+        setTime("");
+        setText("");
+        setCheck(true);
+    }
+
+
+
+
   return (
     <>
-      <h1>Fruitmand bezorgservice</h1>
+      <h1>Fruit basket delivery system</h1>
+
+
+        <FruitComponent
+            fruitText={"Bananas 🍌"}
+            counter={"Bananas:"}
+            fruit={bananas}
+            increasement={() => {
+                minOne(bananas, setBananas)
+            }}
+            decreasement={() => {
+                addOne(bananas, setBananas)
+            }}
+            btnOne={"-1"}
+            btnTwo={"+1"}>
```

Comment:
Zelf had ik dit niet instelbaar gemaakt, maar gewoon opgenomen in het component zelf